### ISSUE TITLE
AI_Error 'can_build_colony_for_species' is forbidden before InitializerLock('colony_builder')

### ISSUE THREAD

**Issue by Oberlus** [freeorion/freeorion#3757]
Bug Report
==========

Environment
-----------

* **FreeOrion Version**: 7c00fa7
* **Operating System**: Ubuntu 20.04
  * Compiled from source

Description
-----------
Got errors in AI at game start:

```
Begin Turn 1
[...]
[21 Mar 13:06:08] AI_1: Unity (Maniacal): Worthless insects, you dare to challenge our mighty empire?!
[21 Mar 13:06:08] AI_1: AI_Error: AI script error : FreeOrionAI.py:generateOrders():367  - Exception Call of 'can_build_colony_for_species' is forbidden before InitializerLock('colony_builder') is initialized while trying to survey_universe
Traceback (most recent call last):
  File "/home/lucas/Projects/FreeOrion/freeorion/default/python/AI/FreeOrionAI.py", line 364, in generateOrders
    action()
  File "/home/lucas/Projects/FreeOrion/master-build/default/python/common/listeners.py", line 40, in wrapper
    res = funct(*args)
  File "/home/lucas/Projects/FreeOrion/freeorion/default/python/AI/ColonisationAI.py", line 197, in survey_universe
    _print_empire_species_roster()
  File "/home/lucas/Projects/FreeOrion/freeorion/default/python/AI/ColonisationAI.py", line 545, in _print_empire_species_roster
    can_build_colony_for_species(species_name),
  File "/home/lucas/Projects/FreeOrion/freeorion/default/python/AI/freeorion_tools/lazy_initializer.py", line 28, in wrapper
    raise ValueError(
ValueError: Call of 'can_build_colony_for_species' is forbidden before InitializerLock('colony_builder') is initialized
Local context: {'args': ('SP_LAENFA',), 'kwargs': {}, 'function': <function can_build_colony_for_species at 0x7f80263b5790>, 'self': <freeorion_tools.lazy_initializer.InitializerLock object at 0x7f80263af970>}

[21 Mar 13:06:08] AI_1: AI_Error: AI script error : FreeOrionAI.py:generateOrders():367  - Exception Call of 'can_build_colony_for_species' is forbidden before InitializerLock('colony_builder') is initialized while trying to calculate_priorities
Traceback (most recent call last):
  File "/home/lucas/Projects/FreeOrion/freeorion/default/python/AI/FreeOrionAI.py", line 364, in generateOrders
    action()
  File "/home/lucas/Projects/FreeOrion/freeorion/default/python/AI/PriorityAI.py", line 63, in calculate_priorities
    ColonisationAI.get_colony_fleets()  # sets aistate.colonisablePlanetIDs and many other values used by other modules
  File "/home/lucas/Projects/FreeOrion/freeorion/default/python/AI/ColonisationAI.py", line 260, in get_colony_fleets
    if not (planet2 and can_build_colony_for_species(planet2.speciesName)):
  File "/home/lucas/Projects/FreeOrion/freeorion/default/python/AI/freeorion_tools/lazy_initializer.py", line 28, in wrapper
    raise ValueError(
ValueError: Call of 'can_build_colony_for_species' is forbidden before InitializerLock('colony_builder') is initialized
Local context: {'args': ('',), 'kwargs': {}, 'function': <function can_build_colony_for_species at 0x7f80263b5790>, 'self': <freeorion_tools.lazy_initializer.InitializerLock object at 0x7f80263af970>}

[21 Mar 13:06:08] AI_1: AI_Error: AI script error : FreeOrionAI.py:generateOrders():367  - Exception Call of 'can_build_only_sly_colonies' is forbidden before InitializerLock('colony_builder') is initialized while trying to generate_production_orders
Traceback (most recent call last):
  File "/home/lucas/Projects/FreeOrion/freeorion/default/python/AI/FreeOrionAI.py", line 364, in generateOrders
    action()
  File "/home/lucas/Projects/FreeOrion/freeorion/default/python/AI/ProductionAI.py", line 133, in generate_production_orders
    _first_turn_action()
  File "/home/lucas/Projects/FreeOrion/freeorion/default/python/AI/ProductionAI.py", line 103, in _first_turn_action
    if can_build_only_sly_colonies():
  File "/home/lucas/Projects/FreeOrion/freeorion/default/python/AI/freeorion_tools/lazy_initializer.py", line 28, in wrapper
    raise ValueError(
ValueError: Call of 'can_build_only_sly_colonies' is forbidden before InitializerLock('colony_builder') is initialized
Local context: {'args': (), 'kwargs': {}, 'function': <function can_build_only_sly_colonies at 0x7f80263b59d0>, 'self': <freeorion_tools.lazy_initializer.InitializerLock object at 0x7f80263af970>}
```

To reproduce, start a game with at least one AI.

**Comment by Oberlus** [comment]
AI is completely crippled. It won't do anything except scouting with the two initial scouts.

**Comment by Morlic-fo** [comment]
Should be fixed by #3755
